source/install/troubleshooting.rst

```diff
@@ -133,6 +133,11 @@ Multi-Factor Authentication (MFA)
 
 If MFA has stopped working, confirm that your server clock is not out of sync. On Unix systems you can view the hardware time of your system using ``# hwclock -r``. If you're using systemd you can also use ``# timedatectl``.
 
+config.json Resets to Default Values
```

Comment:
Actually, I might have been a bit hasty with this one without actually verifying it myself. I am trying to reproduce this but unable to. I was thinking that running the binary as root will take a different config file location, but that's not what's happening from what I can see.